examples/abitest/config/build.sh

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env bash
+
+readonly DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+readonly MODULES_DIR="${DIR}/../../target/wasm32-unknown-unknown/release"
+
+bazel run //oak/common:app_config_serializer -- \
+    --textproto="${DIR}/config.textproto"\
+    --modules="frontend_module:${MODULES_DIR}/abitest_0_frontend.wasm","backend_module:${MODULES_DIR}/abitest_1_backend.wasm" \
+    --output_file="${DIR}/config.bin"
```

Comment:
I think we should probably start considering a dedicated output directory for build artifacts (e.g. `/out`?) Otherwise it's pretty hard to figure out what's source and what's generated (sure, the gitignore file gives you some ideas, but it's not great). So this may be `/out/abitest.bin`, which seems also a bit more self-explanatory.

Reply:
Good point, I was thinking about it. We already have a billion output directories and out sounds generic, but I'll try to find something that makes sense

Reply:
@tiziano88 - the other annoying thing is that afaict, Cargo doesn't allow you to specify a different path for the output (or a link to that). There are a couple of in progress issues, and an experimental feature, but otherwise it sucks. It would be nice to have a /bin/ folder for each example only with the binary artefacts needed to run it